.github/workflows/test.yml

```diff
@@ -11,15 +11,21 @@ jobs:
 
     strategy:
       matrix:
-        php: [ '7.4', '8.0', '8.1' ]
-        laravel: [ '8', '9', '10' ]
+        php: [ '7.4', '8.0', '8.1', '8.2' ]
+        laravel: [ '8', '9', '10', '11' ]
```

Comment:
Is there anything compatible with PHP 8.3? We can test it against 8.3 I think

Reply:
Probably no harm adding 8.3 -- I've pushed that change up.